src/annotator/anchoring/match-quote.ts

```diff
@@ -66,22 +59,29 @@ function textMatchScore(text, str) {
   return 1 - (matches[0].errors / str.length);
 }
 
+type Context = {
+  /** Expected text before the quote */
+  prefix?: string;
+  /** Expected text after the quote */
+  suffix?: string;
+  /** Expected offset of match within text */
+  hint?: number;
+};
+
 /**
  * Find the best approximate match for `quote` in `text`.
  *
- * Returns `null` if no match exceeding the minimum quality threshold was found.
- *
  * @param {string} text - Document text to search
  * @param {string} quote - String to find within `text`
- * @param {object} context -
- *   Context in which the quote originally appeared. This is used to choose the
- *   best match.
- *   @param {string} [context.prefix] - Expected text before the quote
- *   @param {string} [context.suffix] - Expected text after the quote
- *   @param {number} [context.hint] - Expected offset of match within text
- * @return {Match|null}
+ * @param context - Context in which the quote originally appeared. This is
```

Comment:
You can remove the redundant type hint for the preceding `@param`s.

Reply:
Right!